PLCrashReporter.podspec

```diff
@@ -1,4 +1,5 @@
 Pod::Spec.new do |spec|
+  spec.cocoapods_version = '>= 1.10'
```

Comment:
Do we need to update the changelog that we raised the minimal version of cocoapods?
Also please update the template in PR description.